bake/hcl_test.go

```diff
@@ -620,3 +620,79 @@ func TestHCLBuiltinVars(t *testing.T) {
 	require.Equal(t, "foo", *c.Targets[0].Context)
 	require.Equal(t, "test", *c.Targets[0].Dockerfile)
 }
+
+func TestCombineHCLAndJSON(t *testing.T) {
```

Comment:
Please also add tests that actually check the attributes feature with multiple files, not only the reserved blocks.

Reply:
Last commit adds support for attributes too.